### DISCUSSION TITLE
Станок ТВ7

### DISCUSSION THREAD

**Discussion by Kosnov** [kachurovskiy/nanoels#223] [General]
Максим доброго. За проект - благодарности нет предела. Все разобрался, работает. 
Вопрос. 
Станок ТВ7, может видели. У него коробка подач с одним входным валом и на выходе два вала- один ходовой вал и ходовой винт.
Подключив шаговый двигатель на вход- будет ли работать такая связка.

**Comment by kachurovskiy** [comment]
Добрый вечер, @Kosnov. Контроллеру важно лишь на сколько двигается режущий инструмент за 1 шаг мотора. Это значение должно быть постоянным. Наверное, будет?
